.github/workflows/ci.yml

```
@@ -260,3 +260,77 @@ jobs:
       # when imported in an NPM node_modules tree.
       - name: Prepack packages
         run: yarn lerna run --reject-cycles --concurrency 1 prepack
+
+  test-xs:
+    name: test-xs
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20.x]
+        # may include release tags and hashes:
+        moddable-version: [5.0.0]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

I'm not sure if you were aware of this, but `actions/checkout@v4` has the following default for `path':

```yaml
    # Relative path under $GITHUB_WORKSPACE to place the repository
    path: ''
```

So this means the checkouts you do later with `path: moddable` are checking out to `$GITHUB_WORKSPACE/moddable`.

If you meant for the endo and moddable checkouts to be siblings (as I understand @mhofman was suggesting) instead of `moddable` being a subdirectory of the `endo` checkout, you need something like:

```suggestion
        uses: actions/checkout@v4
        path: endo
```

I'll make further suggestions for changes along that line.  If the status quo is sufficient, feel free to ignore those suggestions.

---

I see.